knowledge-base/how-to-programmatically-create-a-master-detail-report-using-subreport-item.md

```diff
@@ -0,0 +1,178 @@
+---
+title: How To: Programmatically Create a Master-Detail Report Using SubReport Item
+description: This article shows how to create a Master-Detail Report using a SubReport Item using only code.
```

Comment:
Change the description so it does not include "This article shows".  For example, "Create a Master-Detail Report using a SubReport item using only code."

Reply:
all set